config/kubernetes/org.yaml

```diff
@@ -558,6 +558,7 @@ members:
 - jmhbnz
 - jmickey
 - jmyung
+- joaquimrocha
```

Comment:
@joaquimrocha, please split this change into a separate comment reading:

`Add joaquimrocha to kubernetes org`

Reply:
Thank you @Priyankasaggu11929 . Done.